Eastern-Research-Group/lew
# Feature/upgrade 3rd party packages

## Related Issues:
* #99 
* #101 

## Main Changes:
* Upgraded the node engine to the latest version.
* Upgraded 3rd party packages. 
  * I followed HMW as a model for husky and helmet.
* Fixed some of the code smells.

## Steps To Test:
1. Run `npm run test`
2. Verify the tests pass
3. Start the application
4. Navigate to http://localhost:9090/
5. Try to test as many features as possible
